.ci_support/osx_64_.yaml

```diff
@@ -16,18 +14,19 @@ cxx_compiler_version:
 - '14'
 gmp:
 - '6'
+libboost_devel:
+- '1.82'
 libflint:
 - '2.9'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 pin_run_as_build:
-  boost-cpp:
-    max_pin: x.x.x
   python:
     min_pin: x.x
     max_pin: x.x
 python:
 - 3.10.* *_cpython
+- 3.11.* *_cpython
```

Comment:
Strangely, there is no Python 3.11 build of cppyy. It seems that the Python 3.11 migration never made it to cppyy, even though its dependencies seem to have Python 3.11 builds.